translations/messages.fr.xlf

```diff
@@ -3766,6 +3789,34 @@
         <source>Simple step</source>
         <target>Simple pas</target>
       </trans-unit>
+      <trans-unit id="3700643537935262310" datatype="html">
+        <source>One of the two kinds of move you can do is the duplication. When you do one, you create a new piece. To do this, select one of your pieces, and click on one of its neighboring space.&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, make a duplication.</source>
+        <target>Un des deux types de mouvement que vous pouvez faire est la duplication. Quand vous en faites un, vous créez une nouvelle pièce. Pour ce faire, sélectionnez une de vos pièces, et cliquez sur une de vos pièces, puis sur une case voisine à celle-ci.&lt;br/&gt;Vous jouez Foncé, faite un simple pas.</target>
+      </trans-unit>
+      <trans-unit id="2864359495844920670" datatype="html">
+        <source>This was a jump, try to do a duplication.</source>
+        <target>C'était un saut, essayez de faire une duplication.</target>
+      </trans-unit>
+      <trans-unit id="4273222196833633277" datatype="html">
+        <source>This was a duplication, try a jump now.</source>
+        <target>C'était une duplication, essayez un saut maintenant.</target>
+      </trans-unit>
+      <trans-unit id="1771050228270959005" datatype="html">
+        <source>The second type of move you can do is the jump. When you do one, your piece leaves its original space and jumps two spaces further. To do this, select one of your pieces, and click on its landing space two spaces further.&lt;br/&gt;&lt;br/&gt;You&apos;re playing Light, make a jump.</source>
+        <target>Le deuxième type de mouvement que vous pouvez faire est un saut. Quand vous en faites un, votre pièce quitte sa case d'origine et atterit deux cases plus loin. Pour faire ceci, sélectionnez une de vos pièces, et cliquez sur sa case d'atterrissage, deux cases plus loin.&lt;br/&gt;Vous jouez Clair, faites un saut.</target>
+      </trans-unit>
+      <trans-unit id="6394015257472015389" datatype="html">
+        <source>When one of your pieces lands on a square that neighbors an opponent&apos;s pieces, those pieces become yours.&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, do such a move!</source>
```

Comment:
an opponent's pieces -> pieces of the opponent

Reply:
"When one of your pieces lands on a square, its adjacent opponent pieces become yours." jeune homme